src/test/java/com/hongliangjie/fugue/topicmodeling/LDA/LDATest.java

```diff
@@ -135,14 +135,76 @@ protected void sampleTestDoc(List<Document> docs, int maxIter, int docIndex) {
         }
     }
 
+    private class DeepHyperOptLDA extends LDA{
+
+        protected class TestProcessor extends ProcessDocuments{
+
+            TestProcessor(Sampler g, HyperparameterOptimization h){
+                super(g, h);
+            }
+
+            private void CheckHyper(int modelID){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/hongliangjie/Fugue/file/2346060966/issues/source?bid=3162994&fileBranchId=3208469#l146)